src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -78,6 +82,9 @@ public abstract class BatchExportProcessor<T> : BaseExportProcessor<T>
                 Name = $"OpenTelemetry-{nameof(BatchExportProcessor<T>)}-{exporter.GetType().Name}",
             };
             this.exporterThread.Start();
+
+            this.exporterName = exporter.GetType().Name;
+            this.processorType = typeof(T).Name;
```

Comment:
Is this resulting in the type of the processor or the type of the data (being exported)?

Reply:
The type of data being exported i.e. `Activity` or `LogRecord`

Reply:
Consider changing the variable name to reflect the intention, e.g. "this.processorType" -> "this.theTypeOfDataBeingExported" (something simpler would be even better).

Reply:
Renamed to `exportedDataType`